set.mm

```diff
@@ -85113,6 +85138,76 @@ the first case of his notation (simple exponentiation) and subscript it
     ( cmap co wcel wf cvv wa wb elmapex elmapg syl ibi ) ABCDEFZCBAGZOBHFCHFIOP
     JABCKBCAHHLMN $.
 
+  ${
+    $d A f m $.  $d B f m $.  $d V m $.
+    $( The set exponentiation ` ( B ^m A ) ` is the set of all functions that
+       map from ` A ` to ` B ` .  Generalisation of ~ mapvalg for arbitrary
+       classes as domains instead of sets only.  (Contributed by AV,
+       8-Aug-2024.) $)
+    mapfset $p |- ( B e. V -> { f | f : A --> B } = ( B ^m A ) ) $=
+      ( vm wcel cv wf cab cmap co vex feq1 elab cvv simpr fdmexb mpbiri adantr
+      wa elmapd exbiri pm2.43b elmapi impbid1 syl5bb eqrdv ) BDFZEABCGZHZCIZBAJ
+      KZEGZUKFABUMHZUHUMULFZUJUNCUMELZABUIUMMNUHUNUOUHUNUOUNUHUOUNUNUHTBAUMDOUN
+      UHPUNAOFZUHUNUQUMOFUPABUMQRSUAUBUCUMBAUDUEUFUG $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The set exponentiation ` ( B ^m A ) ` is a subset of the class of
+       functions that map from ` A ` to ` B ` .  (Contributed by AV,
+       10-Aug-2024.) $)
+    mapssfset $p |- ( B ^m A ) C_ { f | f : A --> B } $=
+      ( cvv wcel cmap co cv wf cab wss wceq mapfset eqimss2 syl reldmmap ovprc1
+      wn c0 0ss eqsstrdi pm2.61i ) BDEZBAFGZABCHICJZKZUCUEUDLUFABCDMUDUENOUCRUD
+      SUEBAFPQUETUAUB $.
+
+    $d A f m $.  $d B m $.
+    $( The set exponentiation ` ( B ^m C ) ` is a superset of the set of all
+       functions that map from ` A ` onto ` B ` .  (Contributed by AV,
+       7-Aug-2024.) $)
+    mapfoss $p |- { f | f : A -onto-> B } C_ ( B ^m A ) $=
+      ( vm cv wfo cab cmap co wcel vex foeq1 elab wf fof cvv crn forn eqeltrrdi
+      rnex fdmexb mpbiri syl elmapd mpbird sylbi ssriv ) DABCEZFZCGZBAHIZDEZUJJ
+      ABULFZULUKJZUIUMCULDKZABUHULLMUMUNABULNZABULOZUMBAULPPUMBULQPABULRULUOTSU
+      MUPAPJZUQUPURULPJUOABULUAUBUCUDUEUFUG $.
+  $}
+
+  ${
+    $d A f $.
+    $( The set of functions with a proper class as domain is empty.
+       (Contributed by AV, 22-Aug-2024.) $)
+    fsetdmprc0 $p |- ( A e/ _V -> { f | f Fn A } = (/) ) $=
+      ( cvv wnel cv wfn wn wal cab c0 wceq wcel df-nel vex a1i id fndmexd con3i
+      sylbi alrimiv ab0 sylibr ) ACDZBEZAFZGZBHUEBIJKUCUFBUCACLZGUFACMUEUGUEAUD
+      CUDCLUEBNOUEPQRSTUEBUAUB $.
+
+    $d B f $.
+    $( The set of functions between two classes exists if the codomain exists.
+       Generalization of ~ mapex for arbitrary classes as domains instead of
+       sets only.  (Contributed by AV, 14-Aug-2024.) $)
```

Comment:
```suggestion
       Generalization of ~ mapex to arbitrary domains.  (Contributed by AV, 14-Aug-2024.) $)
```

Also, new readers may be confused by this result, so I would add:
```
Note that the class ` { f | f : A --> B } ` can only contain set-functions, as opposed to
arbitrary class-functions.  When ` A ` is a proper class, there can be no set-functions
on it, so the above class is empty, hence a set.
```
With maybe in the other three results: `See also comment of ~ fsetex .`

Reply:
I will add this note to the comment of ~mapfset.